curriculum/challenges/english/blocks/workshop-fortune-teller-app/68c38c3275a149b3cfed312a.md

```diff
@@ -0,0 +1,400 @@
+---
+id: 68c38c3275a149b3cfed312a
+title: Step 1
+challengeType: 0
+dashedName: step-1
+demoType: onLoad
+---
+
+# --description--
+
+In this Workshop, you are going to build a fortune-telling-app step by step.
```

Comment:
workshop doesn't need be capitalized here and we don't need to mention step by step since campers will already be familiar the structure of workshops by now 👍🏾 

```suggestion
In this workshop, you will continue to practice working with TypeScript by building a fortune telling app.
```